deployment/secure_research_environment/setup/Update_Stored_Access_Policies.ps1

```diff
@@ -0,0 +1,38 @@
+param(
+    [Parameter(Mandatory = $true, HelpMessage = "Enter SHM ID (e.g. 'project')")]
+    [string]$shmId,
+    [Parameter(Mandatory = $true, HelpMessage = "Enter SRE ID (e.g. 'sandbox')")]
+    [string]$sreId
+)
+
+Import-Module Az.Accounts -ErrorAction Stop
+Import-Module Az.Resources -ErrorAction Stop
+Import-Module Az.Storage -ErrorAction Stop
+Import-Module $PSScriptRoot/../../common/AzureStorage -Force -ErrorAction Stop
+Import-Module $PSScriptRoot/../../common/Configuration -Force -ErrorAction Stop
+Import-Module $PSScriptRoot/../../common/DataStructures -Force -ErrorAction Stop
+Import-Module $PSScriptRoot/../../common/Logging -Force -ErrorAction Stop
+
+
+# Get config and original context before changing subscription
+# ------------------------------------------------------------
+$config = Get-SreConfig -shmId $shmId -sreId $sreId
+$originalContext = Get-AzContext
+$null = Set-AzContext -SubscriptionId $config.sre.subscriptionName -ErrorAction Stop
+
+# Update each SAS Access Policy to be valid for one year from now
```

Comment:
Should the default here (1 more year) be different from the original default (20 years)?